src/routes/houses.routes.js

```diff
@@ -33,6 +39,16 @@ const housesRoutes = (router) => {
 
     return res.status(StatusCodes.OK).json(response);
   });
+
+  router.patch('/create-new-house/:id', auth, async (req, res) => {
```

Comment:
I know you're just using an existing name here but I haven't noticed this before - it would be good if all houses endpoints were named just `houses` and their purpose differentiated by a HTTP method used. That's something to consider in the future.